generators/vsi/templates/terraform/scripts/build_master.sh

```diff
@@ -20,3 +20,38 @@ mvn clean install
 mvn clean install
 target/liberty/wlp/bin/server package defaultServer --archive="{{deployment.name}}" --include=minify
 {{/has}}
+{{#has deployment.language 'GO'}}
+# get project name from manifest.yml
+export project_name="$(cat manifest.yml | grep -w "GOPACKAGENAME :" | sed 's/    GOPACKAGENAME : //')"
+# get the original folder name from the pwd command 
+original_folder_name="$(echo $(pwd) | sed 's/\/home\/pipeline\///')"
+# go into the directory above the folder w/ all the go files 
+cd /home/pipeline
+# zip the entire project  
+tar -czvf zippedProject.tar.gz $original_folder_name
```

Comment:
do we have to `tar` the project just to unzip it later at line 39? I believe you said that you couldn't `mv` over all of the files at once... Would using `cp -R` (recursively copying) work? That would shorten/remove several lines of code. (I think it would remove lines 29, 31, 35, 39, 40)